uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/PopoverTextCell.java

```diff
@@ -37,6 +39,8 @@
  */
 public class PopoverTextCell extends AbstractSafeHtmlCell<String> {
 
+    protected static final CommonCss css = CommonResources.INSTANCE.CSS();
```

Comment:
constant naming should be upper case and words divided by underscore:
css -> CSS

Reply:
Done